### ISSUE TITLE
Audit logging levels

### ISSUE THREAD

**Issue by ijemafe** [OpenRVS-devs/OpenRVS#42]
Once we finish our current arc of work, and we are ready to release the next version of OpenRVS, we should review every place we call OpenLogger and make sure we're choosing the right level of visibility (specifically, Debug vs non-Debug). We should think about which logs we'll want to get from users to help them troubleshoot problems.

**Comment by ijemafe** [comment]
Alternatively, we can have a config var in OpenLogger called `DebugMode=False`. Then, in `openrvs.ini`, the user can set `DebugMode=True` if we need extra logs from them.

**Comment by rvstwi** [comment]
I agree, however, if we're using a static function to log, we can't use config variables (or any variables other than arguments passed to the function, consts, and maybe vars in the default properties).  This is because the class with the static function doesn't actually exist as an object in game (we don't need to create it with new() or spawn()).

I think the advantages of a static function outweigh the disadvantages in this case.  But it means that there isn't a simple config way of toggling debug mode.
